x-pack/filebeat/docs/inputs/input-azure-eventhub.asciidoc

```diff
@@ -0,0 +1,69 @@
+[role="xpack"]
+
+:type: azure-eventhub
+
+[id="{beatname_lc}-input-{type}"]
+=== Azure eventhub input
+
+++++
+<titleabbrev>Azure eventhub</titleabbrev>
+++++
+
+
+Use the `azure-eventhub` input to read messages from an azure eventhub.
```

Comment:
Could you add some more explanation on how it works and how it uses the storage account? I see some references at the end of the document, but more info here would be helpful

Reply:
I have updated the doc, let me know if more is necessary